docs/ref/urls.txt

```diff
@@ -5,7 +5,21 @@
 .. module:: django.urls.conf
     :synopsis: Functions for use in URLconfs.
 
-.. currentmodule:: django.conf.urls
+.. currentmodule:: django.urls
+
+``path()``
+==========
+
+.. function:: path(regex, view, kwargs=None, name=None)
```

Comment:
Should the `regex` parameter be renamed since it's not a regex in this case?

Reply:
I think calling the parameter `route` for both `path()` and `re_path()` could be fine. The only annoying thing for upgraders could be if a `url()` is passing `regex` as a keyword argument. Doing that seems unexpected though.

Reply:
If we leave the django.conf.urls in place, that could still use regex, and then use route for re_path.